src/result_processor.h

```diff
@@ -132,6 +132,9 @@ static inline RSValue *SearchResult_GetValue(SearchResult *res, RSSortingTable *
 // EOF - no results from this processor
 #define RS_RESULT_EOF 2
 
+// STOP - Abort processing immediately!
+#define RS_RESULT_STOP -1
```

Comment:
Remove this please. we are assuming that we either have ok/eof/err on so many places, do not introduce another return code. use the query state instead.

Reply:
yeah, this is a leftover.